module/arm5e.js

```diff
@@ -202,6 +199,11 @@ Hooks.once("init", async function () {
     }
     return options.inverse(this);
   });
+
+  Handlebars.registerHelper("isGM", function () {
```

Comment:
Yeah I put `isGM` in the `getData` context too but started using handlebar helpers more and I like it there (only for highly general usages though, anything specific to a single thing should just be handled in `getData`

It is indeed `{{#if (isGM)}}` but you never have to bother putting it in `getData`, I use it here:
https://github.com/Xzotl42/arm5e/pull/284/files#diff-26a88eb5f8b987d04324e90a8e631b46e753b151b27860b5462f0779582ee3d4R93

Here's more than I find helpful, if they might help you as well


```js
    Handlebars.registerHelper("config", function (key) 
    {
        return IMPMAL[key];
    });
```
e.g.
```html
{{#each (config "skills")}}
<!-- Display skill data -->
{{/each}}
```

```js
    Handlebars.registerHelper("configLookup", function (obj, key) 
    {
        return getProperty(IMPMAL[obj], key);
    });
 ```
 e.g.
```html
{{configLookup "skills" "label"}}
```



Easily access setting value
```js
    Handlebars.registerHelper("settings", function (key) 
    {
        return game.settings.get("impmal", key);
    });
```




Create percentage, useful for progress bars 
```js
    Handlebars.registerHelper("pct", function (part, whole, max100=true) 
    {
        let pct =  (part / whole) * 100;
        if (pct > 100 && max100)
        {
            pct = 100;
        }
        return pct;
    });
```
e.g.
```html
<div class="bar" style="width : {{pct system.encumbrance.value system.encumbrance.max}}%"></div>
```

Lastly, I use token names and token images as the "player friendly" versions, where if a GM doesn't want to reveal their identity, they set the token name/image accordingly and everything displayed regarding them uses that, however it was difficult because some things are real actors, some things are synthetic, so this helper fixes that.

```js
    Handlebars.registerHelper("tokenImg", function (actor) 
    {
        try 
        {
            return actor.token ? actor.token.texture.src : actor.prototypeToken.texture.src;
        }
        catch (e) 
        {
      
        }
    });

    Handlebars.registerHelper("tokenName", function (actor) 
    {
        try 
        {
            return actor.token ? actor.token.name : actor.prototypeToken.name;
        }
        catch (e) 
        { 
     
        }
    });
```
e.g.
```html
<img src="{{tokenImg attacker.test.actor}}">
```

Reply:
Thanks, those will be helpful, I like especially the last one